### PR TITLE
 time picker updated

### PR DESCRIPTION
**Pull request by sayerwalam52**

### CODE REVIEW

File: config/database.yml [hunk 1]
```diff
@@ -17,6 +17,8 @@
 default: &default
   adapter: postgresql
   encoding: unicode
+  username: postgres
```
Comment (anagabriel42) [inline config/database.yml]: remove these plz

Comment (sayerwalam52) [reply config/database.yml]: done

File: app/views/settings/_settings_auto_form.html.erb [hunk 2]
```diff
@@ -9,12 +9,26 @@
 
     <div class='col-8'>
 
+
       <div class='form-group fg-line time'>
```
Comment (anagabriel42) [inline app/views/settings/_settings_auto_form.html.erb]: starting here

File: app/views/settings/_settings_auto_form.html.erb [hunk 3]
```diff
@@ -9,12 +9,26 @@
...
+              <li class="next noselect"> + </li>
+              <li id="seconds"> </li>
+              <li class="prev noselect"> - </li>
+            </ul>
+          </div>
+        </div>
+
       </div>
 
     </div>
```
Comment (anagabriel42) [inline app/views/settings/_settings_auto_form.html.erb]: to here

File: app/views/settings/_settings_auto_form.html.erb [hunk 4]
```diff
@@ -33,8 +47,42 @@
     <%= f.submit 'update', class: 'form-button' %>
   </div>
 
-<% end %>
-
 <script>
```
Comment (anagabriel42) [inline app/views/settings/_settings_auto_form.html.erb]: and here

File: app/views/settings/_settings_auto_form.html.erb [hunk 5]
```diff
@@ -33,8 +47,42 @@
...
+  $(".prev").on("click", function() {
+        if (seconds >= 1){
+        seconds--
+      $("#seconds").text(seconds);
+      $("#time").val(seconds);
+      }else {
+        alert ("Can't go negative");
+      }
+  });
 </script>
```
Comment (anagabriel42) [inline app/views/settings/_settings_auto_form.html.erb]: to here

**Pull request merged**